policy/release/authorization.rego

```diff
@@ -30,13 +30,14 @@ deny[result] {
 #   Enterprise Contract verifies if an authorized commit was used as the source of a build
 # custom:
 #   short_name: disallowed_commit_does_not_match
-#   failure_msg: Commit %s does not match authorized commit %s
+#   failure_msg: Commit %s does not match authorized commits
 deny[result] {
 	data.authorization
+	count(data.authorization) > 0
 	att := lib.pipelinerun_attestations[_]
 	material := att.predicate.materials[_]
-	data.authorization.changeId != material.digest.sha1
-	result := lib.result_helper(rego.metadata.chain(), [material.digest.sha1, data.authorization.changeId])
+	not sha_in_auth(material.digest.sha1, data.authorization)
```

Comment:
(nit) could be done without the helper method

```suggestion
	not material.digest.sha1 in [c | a = data.authorization[_]; c = a.changeid]
```

Reply:
For some reason I get `unexpected ident token: expected \n or ; or }` when I add this.

Reply:
Odd, I [tried](https://play.openpolicyagent.org/p/UlT1jRG3fO) it in the playground, it’s really a nit, no need to block the merge

Reply:
Even more odd that it doesn't complain when I add the filter to the other function.

Reply:
Let’s merge, we can refactor when/if needed